app/controllers/api/matches_controller.rb

```diff
@@ -2,21 +2,11 @@ module API
   class MatchesController < ApplicationController
     # @route POST /api/matches (api_matches)
     def create
-      @match = Match.new(match_params)
-      @match.match_players.each do |match_player|
-        existing_player = Player.find_by(name: match_player.player.name)
-
-        next unless existing_player.present?
-
-        elo_change = match_player.elo_change
-        match_player.player = existing_player
-        match_player.elo_change = elo_change
-      end
-
-      if @match.save
-        render json: @match.attributes.merge(match_url: match_url(@match)), status: :created
+      match_data_form = MatchDataForm.new(match_params: match_params, scoreboard_log_params: scoreboard_log_params)
```

Comment:
Para qué necesitamos un form object acá? 

Se me hace que ahora con los PlayerStats, ya no es necesario el form object o si? 
- No hay un form en el cliente
- Parece que MatchDataForm no tiene validaciones ni callbacks

Reply:
Me gustaba mantener el form object, porque saca logica del controller. That being said, puedo cambiar y eliminar definiticamente el form object, el repo is yours, no quiero entrometerme mucho tampoco 😅

Reply:
yo lo dejaría todo en el controller por simplicidad, como que no hay mucha logica tampoco o si? O qué lógica nueva se agrega acá? creo que lo único que se agregó es lo del player_stat_attributes en los strong params no?